source/jormungandr/jormungandr/parking_space_availability/bss/tests/bss_provider_manager_test.py

```diff
@@ -85,6 +85,28 @@ def realtime_place_handle_test():
     places_with_stands = manager.handle_places(places)
     assert 'stands' in places_with_stands[0]['poi']
 
+def realtime_place_handle_test():
```

Comment:
hum take care, you use the same method name as the previous one, python will give you no warning, but only one will be used :wink: